### DISCUSSION TITLE
Custom Rules Detection

### DISCUSSION THREAD

**Discussion by savidudu** [CrowdStrike/community#71] [Q&A]
Hello All,

Do we have something available for mac machines similar to below which is for Windows ?

https://www.crowdstrike.com/blog/tech-center/custom-ioas/

I am trying to check but no luck.

If this is the right channel then please help me.

Thanks

**Comment by redhatrises** [comment]
Hello.

Thanks for asking this question. It is best for product specific questions like this one to either reach out to your CS account team or to open a support ticket to get an answer.
